Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM zonemaster/engine:local as build
+
+RUN apk add --no-cache \
+    # Only needed for Net::Interface
+    build-base \
+    perl-dev \
+    perl-doc \
+    perl-lwp-protocol-https \
+    make \
+    # Compile-time dependencies
+    perl-app-cpanminus \
+    perl-module-install \
+ && cpanm --no-wget --from https://cpan.metacpan.org/ \
+    MooseX::Getopt \
+    Text::Reflow
+
+ARG version
+
+COPY ./Zonemaster-CLI-${version}.tar.gz ./Zonemaster-CLI-${version}.tar.gz
+
+RUN cpanm --no-wget \
+    ./Zonemaster-CLI-${version}.tar.gz
+
+FROM zonemaster/engine:local
+
+COPY --from=build /usr/local/bin/zonemaster-cli /usr/local/bin/zonemaster-cli
+# Include all the Perl modules we built
+COPY --from=build /usr/local/lib/perl5/site_perl /usr/local/lib/perl5/site_perl
+COPY --from=build /usr/local/share/perl5/site_perl /usr/local/share/perl5/site_perl
+
+USER nobody:nogroup
+
+ENTRYPOINT [ "zonemaster-cli" ]
+
+CMD [ "--help" ]
```

Comment:
Translation does not work here. Can that be added?

Reply:
In what way does it not work?

This works for me:
```sh
sudo docker run -e LANG=sv_SE zonemaster/cli:local zonemaster.net
```

Reply:
I do not get any translation with that command.

Reply:
I tried the following commands and I can the translation in both cases
```
docker run --rm -it zonemaster/cli:local --locale fr zonemaster.net
docker run -e LANG=fr_FR.UTF-8 --rm -it zonemaster/cli:local zonemaster.net
```

Reply:
It was not documented, but `make all` is required before the image is built. See https://github.com/zonemaster/zonemaster-cli/pull/218#pullrequestreview-798942149

Reply:
I updated the README to include the make all target.